scripts/ubuntu/mongo-orchestration.sh

```diff
@@ -23,11 +23,11 @@ dpkg -x mongodb-enterprise-server_3.2.0_amd64.deb 3.2.0
 dpkg -x mongodb-enterprise-mongos_3.2.0_amd64.deb 3.2.0
 dpkg -x mongodb-enterprise-server_3.6.0_amd64.deb 3.6.0
 
-
 # Python stuff for mongo-orchestration
-apt-get install -y python python-dev python-pip
+apt-get install -y python python-dev
+python get-pip.py
 
-pip install --index-url=https://pypi.python.org/simple/ --upgrade 'git+https://github.com/10gen/mongo-orchestration.git#egg=mongo_orchestration'
+pip install --upgrade mongo-orchestration
```

Comment:
This is the [documented method for installing mongo-orchestration](https://github.com/10gen/mongo-orchestration#installation), provided that pip is up-to-date.

The original command was so verbose because the `python-pip` package on Ubuntu Precise (12.04) is rather ancient.

Reply:
Shouldn't we upgrade our Precise thing to something newer as well then?

Reply:
New commits updates the VM to trusty64.